testsuite/test_pdf_preview.py

```diff
@@ -0,0 +1,54 @@
+import pytest
+from snoop.data.analyzers import pdf_preview
+from conftest import TESTDATA, CollectionApiClient
+from snoop.data import models
+
+pytestmark = [pytest.mark.django_db]
+
+
+def test_pdf_preview_service():
+    test_doc = TESTDATA / './disk-files/word/sample-doc-file-for-testing-1.doc'
```

Comment:
put this filename in a const in this test module, since you're using it in a few tests